src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,57 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp(){
```

Comment:
```suggestion
    void setUp() {
```

메서드에서 공백을 생략하는 것이 승재님의 컨벤션일까요!?
해당 미션을 진행하는 동안은 컨벤션에 맞게 작성해보면 어떨까하는 생각이 있는데 승재님의 의견이 궁금합니다 🤔

Reply:
컨벤션에 맞게 코드를 작성해 보겠습니다!